### ISSUE TITLE
Are tests running on TravisCI ?

### ISSUE THREAD

**Issue by adamchester** [serilog/serilog#777]
It seems like Linux/OSX tests aren’t run during CI builds, is that a known issue?

For example: https://travis-ci.org/serilog/serilog/jobs/138635977#L1696
When I run build.sh on OSX manually, I see this output
```
Project Serilog.Tests (.NETCoreApp,Version=v1.0) was previously compiled. Skipping compilation.
xUnit.net .NET CLI test runner (64-bit osx.10.11-x64)
  Discovering: Serilog.Tests
  Discovered:  Serilog.Tests
  Starting:    Serilog.Tests
  Finished:    Serilog.Tests
=== TEST EXECUTION SUMMARY ===
   Serilog.Tests  Total: 223, Errors: 0, Failed: 0, Skipped: 0, Time: 1.223s
Project Serilog (.NETStandard,Version=v1.3) was previously compiled. Skipping compilation.
Project TestDummies (.NETStandard,Version=v1.2) was previously compiled. Skipping compilation.
```

Output from Travis https://travis-ci.org/serilog/serilog/jobs/138635976#L432
```
Project Serilog.Tests (.NETCoreApp,Version=v1.0) was previously compiled. Skipping compilation.
The specified framework 'Microsoft.NETCore.App', version '1.0.0-rc2-3002702' was not found.
  - Check application dependencies and target a framework version installed at:
      /Users/travis/build/serilog/serilog/.dotnetcli/shared/Microsoft.NETCore.App
  - The following versions are installed:
      1.0.0
  - Alternatively, install the framework version '1.0.0-rc2-3002702'.
Project Serilog (.NETStandard,Version=v1.3) was previously compiled. Skipping compilation.
```

**Comment by merbla** [comment]
Previous builds were ok, probably need a clear exist code.

[Last success with test runs](https://travis-ci.org/serilog/serilog/builds/137131371)

https://travis-ci.org/serilog/serilog/builds

**Comment by merbla** [comment]
This will be fixed up in the `staging-2.0` branch.

**Comment by merbla** [comment]
Fixed in #789
